dask/utils.py

```diff
@@ -2064,3 +2064,20 @@ def maybe_pluralize(count, noun, plural_form=None):
         return f"{count} {noun}"
     else:
         return f"{count} {plural_form or noun + 's'}"
+
+
+def is_namedtuple_instance(obj: Any) -> bool:
```

Comment:
Moved `is_namedtuple_instance` to `dask.utils` since we want to use it in several places and added a docstring. It now checks for the existence of the entire `namedtuple` API, so this should be accurate.